app/src/main/java/de/danoeh/antennapod/dialog/PlaybackControlsDialog.java

```diff
@@ -5,33 +5,27 @@
 import android.os.Handler;
 import android.os.Looper;
 import android.view.View;
+
 import androidx.annotation.NonNull;
 import androidx.appcompat.app.AlertDialog;
 import androidx.fragment.app.DialogFragment;
 import android.widget.Button;
 import android.widget.CheckBox;
-import android.widget.TextView;
 import de.danoeh.antennapod.R;
-import de.danoeh.antennapod.event.playback.SpeedChangedEvent;
 import de.danoeh.antennapod.core.preferences.UserPreferences;
 import de.danoeh.antennapod.core.util.playback.PlaybackController;
-import de.danoeh.antennapod.view.PlaybackSpeedSeekBar;
 import org.greenrobot.eventbus.EventBus;
-import org.greenrobot.eventbus.Subscribe;
-import org.greenrobot.eventbus.ThreadMode;
-
 import java.util.List;
-import java.util.Locale;
 
 public class PlaybackControlsDialog extends DialogFragment {
+
     private PlaybackController controller;
     private AlertDialog dialog;
-    private PlaybackSpeedSeekBar speedSeekBar;
-    private TextView txtvPlaybackSpeed;
 
     public static PlaybackControlsDialog newInstance() {
         Bundle arguments = new Bundle();
         PlaybackControlsDialog dialog = new PlaybackControlsDialog();
+
```

Comment:
This line is unrelated to your PR. Please have a look at the "Files changed" tab on GitHub and try to minimize touching lines that are not related to your PR. That makes the changes easier to read and also helps with tools like `git blame`.